src/ngAnimate/shared.js

```diff
@@ -38,7 +24,7 @@ var CSS_PREFIX = '', TRANSITION_PROP, TRANSITIONEND_EVENT, ANIMATION_PROP, ANIMA
 // Also, the only modern browser that uses vendor prefixes for transitions/keyframes is webkit
 // therefore there is no reason to test anymore for other vendor prefixes:
 // http://caniuse.com/#search=transition
-if (isUndefined(window.ontransitionend) && isDefined(window.onwebkittransitionend)) {
+if ((window.ontransitionend === void 0) && (window.onwebkittransitionend !== void 0)) {
```

Comment:
They are initialized in the config phase. Here they are accessed before that. If we move the initialization of the constant inside a provider or config block, then we need to ensure that everything that depends on them doesn't run before they are available.

I opted for the shortest viable fix :smile: